### ISSUE TITLE
java.lang.NullPointerException on Record.setWithAck

### ISSUE THREAD

**Issue by AlexBHarley** [deepstreamIO/deepstream.io-client-java#75]
```java.lang.NullPointerException: Attempt to invoke virtual method 'boolean io.deepstream.Event.equals(java.lang.Object)' on a null object reference at io.deepstream.DeepstreamClientAbstract.onError(DeepstreamClientAbstract.java:48)```

**Comment by AlexBHarley** [comment]
```
public class test {

    DeepstreamClient client;

    public static void main(String[] args) throws URISyntaxException {
        new test().run();
    }

    public void run() throws URISyntaxException {
        client = new DeepstreamClient("localhost:6020");
        client.login();
        JsonObject params = new JsonObject();
        params.addProperty("name", "test");
        params.addProperty("age", "23");
        SetRecord("user" ,params.toString());
    }

    public void SetRecord( String recordname , String val)
    {
        JsonParser jsonParser = new JsonParser();
        JsonObject data = (JsonObject)jsonParser.parse(val);
        Record rec;
        rec = client.record.getRecord(recordname);
        String r =   rec.setWithAck(data).getResult();
        System.out.println(String.format("Has result: %s", r));
    }
}
```

Reproducible with this

**Comment by AlexBHarley** [comment]
Fixed in #76
